docs/docs/integrations/mqtt.md

```diff
@@ -123,6 +123,11 @@ Topic with current state of snapshots for a camera. Published values are `ON` an
 Topic to turn motion detection for a camera on and off. Expected values are `ON` and `OFF`.
 NOTE: Turning off motion detection will fail if detection is not disabled.
 
+### `frigate/<camera_name>/motion`
+
+Whether camera_name is currently detecting motion. Expected valeus are `ON` and `OFF`.
+NOTE: Will reset to `OFF` after configurable amount of time (30 seconds by default).
```

Comment:
This makes it sound like it resets to off after 30 seconds always. Can you clarify that it resets to off 30 seconds after the last time motion was detected?